zcash_client_sqlite/src/error.rs

```diff
@@ -119,16 +118,16 @@ pub enum SqliteClientError {
     NoteFilterInvalid(NoteFilter),
 
     /// The proposal cannot be constructed until transactions with previously reserved
-    /// ephemeral address outputs have been mined. The parameters are the account UUID and
-    /// the index that could not safely be reserved.
+    /// ephemeral address outputs have been mined. The error contains the index that could not
+    /// safely be reserved.
     #[cfg(feature = "transparent-inputs")]
-    ReachedGapLimit(AccountUuid, u32),
+    ReachedGapLimit(u32),
 
-    /// An ephemeral address would be reused. The parameters are the address in string
-    /// form, and the txid of the earliest transaction in which it is known to have been
-    /// used.
-    #[cfg(feature = "transparent-inputs")]
-    EphemeralAddressReuse(String, TxId),
+    /// The wallet attempted to create a transaction that would use of one of the wallet's
+    /// previously-used addresses, potentially creating a problem with on-chain transaction
```

Comment:
```suggestion
    /// previously-used transparent addresses, potentially creating a problem with on-chain transaction
```

Reply:
I specifically generalized this to `AddressReuse` in order to make it possible to use this for unified addresses that contain a transparent receiver.